src/main/java/core/basesyntax/Application.java

```diff
@@ -2,6 +2,8 @@
 
 public class Application {
     public static void main(String[] args) {
-        // create three balls using class Lottery and print information about them in console
+        System.out.println(Lottery.getRandomBall());
```

Comment:
The use of static methods is discouraged according to the checklist. Consider creating an instance of the Lottery class and then calling the instance method to get random balls.